ansible_collection/hpe/nimble/plugins/module_utils/hpe_nimble.py

```diff
@@ -41,7 +41,8 @@ def get_unique_string(baseName):
 def remove_null_args(**kwargs):
     tosearch = kwargs.copy()
     for key, value in tosearch.items():
-        if type(value) is not bool:
+        # list can be empty in case of update. Hence we should not remove that arg
```

Comment:
fix for bug 156.. list can be provided as empty to remove all  the existing iscsi initiators from acr